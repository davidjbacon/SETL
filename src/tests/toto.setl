eat 'pudding';
rock 'me' 'gently';
print(rock 'this' 666);
drink;

command eat(what);
  print('eat',what);
end eat;

command drink;
  print('drink');
end command drink;

command rock(who,how);
  print('rock',who,how);
  return 69.95;
end;
