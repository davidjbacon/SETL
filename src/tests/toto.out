eat pudding
rock me gently
rock this 666
69.95
drink
